gt-scheduler/website
# Display custom blocks

### Summary

Resolves #153 

Adds custom event blocks to calendar

### Checklist
#### UI Requirements
- [x] The block is placed at the right date and time on the calendar.
- [x] The block has a minimum height, so even when the event's duration is too short, users can at least see the name of the event ([Figma](https://www.figma.com/file/CkystexX4qQO9OnZRxktXa/GT-Scheduler-%2F-Spr21?node-id=3207%3A2125&t=8EcWnMFd70puiz4A-0))
- [x] Name of the event should have a character limit when displayed on `Calendar`. Full name is shown in the popup instead.
- [x] Mobile view works.

#### Functional Requirements
- [x] Popup appears when hovered over and is focused on click.

### How to Test
Add events of varying sizes (10 min, 20 min, 30 min) and varying name lengths.

Notes: 
- There is probably some extraneous code I haven't gotten rid of or property edited from TimeBlock.
- Minimum time block height is set to 15 min.
- Minimum time block length to see time info is set to 30 min.
